src/os.rs

```diff
@@ -115,10 +115,20 @@ fn unix_lang() -> Result<String> {
             Error::new(kind, e)
         })
     };
-    let langs = check_var("LC_ALL")
-        .or_else(|_| check_var("LANGS"))
+
+    // Uses priority defined in
+    // <https://www.gnu.org/software/gettext/manual/html_node/Locale-Environment-Variables.html>
+    let locale = check_var("LC_ALL")
         .or_else(|_| check_var("LANG"))
-        .or_else(|_| check_var("LANGUAGE"))?;
+        .or_else(|_| check_var("LANGS"));
```

Comment:
This variable doesn't appear in the spec, but it might be a nonstandard method to set the language. Should we drop this check or continue to use it?

Reply:
Let's drop the check